docs/source/daml/intro/daml/daml-intro-6/Parties.daml

```diff
@@ -78,6 +81,9 @@ template Iou
 -- PROPOSE_TRANSFER_END
 
 -- MUTUAL_TRANSFER_BEGIN
+    -- This is a choice on the Iou template.
+    -- Note that Iou does not have the signature of newOwner, so this
+    -- must be called in a context where it is available. See text.
```

Comment:
Even if `newOwner` was a signatory of the `Iou` instance, for instance if the `Iou` is moved back to the `issuer`, you still need the authority of `newOwner` to exercise the `Mutual_Transfer` choice in the _surrounding_ context. You also need the `owner`'s authority in that surrounding context, otherwise you could simply steal anybody's `Iou`s. 😺

Reply:
Yes all of that is explained in the text, and there is an example of calling it in a way that works. The goal here is just to have a signpost for people skimming the page and looking mostly at the code samples (which is really easy to do, visually, given our styling & colour scheme). Repeating the entire text would not be very helpful I think.

Reply:
I think the comment is still misleading because nothing would change if the contract had the signature of `newOwner`. If `x` is a controller of a choice, then the context exercising that choice must always contain the authority of `x`, regardless of whether `x` has signed the contract the choice is exercised on or not.

Reply:
Feel free to provide a better one. My point is, when viewed in the context of [the introduction to DAML](https://docs.daml.com/0.13.55/daml/intro/6_Parties.html#use-role-contracts-for-ongoing-authorization), this snippet of code is confusing.